texar/tf/modules/embedders/embedders.py

```diff
@@ -51,7 +51,7 @@ class WordEmbedder(EmbedderBase):
             :meth:`default_hparams` for the hyperparameter structure and
             default values.
 
-    See :meth:`_build` for the inputs and outputs of the embedder.
+    See :meth:`call` for the inputs and outputs of the embedder.
```

Comment:
since we override `__call__` to have customized arguments, while the 1st argument of `call` is still `input` -- we shall expose `__call__` to users, instead of `call`. The readthedocs should include `__call__` and exclude `call`